docs/en/sql-reference/functions/array-functions.md

```diff
@@ -2118,6 +2118,113 @@ Result:
 └─────────────────────┘
 ```
 
+
+## randomSampleFromArray
```

Comment:
Basically *all* functions that process arrays start with `array`. We should rather name this function `arrayRandomSample()`.

Reply:
It makes sense. I will fix it.

Reply:
done